test/integration/proposals_config.ts

```diff
@@ -16,23 +17,29 @@ const proposals: ProposalsConfigMap = {
         proposal: fip_xx_proposal // full proposal file, imported from '@proposals/description/fip_xx.ts'
     }
     */
-  peg_stability_module: {
+  backstop: {
     deploy: true,
     skipDAO: false,
     totalValue: 0,
-    proposal: peg_stability_module
+    proposal: backstop_proposal
   },
-  backstop: {
+  peg_stability_module: {
     deploy: true,
     skipDAO: false,
     totalValue: 0,
-    proposal: backstop_proposal
+    proposal: peg_stability_module
```

Comment:
weird diff, you inverted the position of the two objects ?

Reply:
Maybe the linter made this happen?